README.md

```diff
@@ -25,3 +25,119 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
+
+A recurrence relation for Divide and Conquer's T(n) can be provided similar to that as was done for mergesort within the slides. 
+For refrence, mergesort's recurrence relation is $T(n) = 2T(n/2) + n$.
+This is due to behavior of mergesort, where the array is split into two halves, each sorted recursively, and then merged, resulting in proper sorting mergesort implementation.
+This the example that was followed and solved within the slides using substitution.
+
+For the divideAndConquerSum function, the recurrence relation is instead $T(n) = 3T(n/3) + c$, reflecting the division into three subarrays and the constant time (expressed by constant c) required to combine their sums. 
+The initial relation for my Divide and Conquer implementation is given this way because at each step, the array is divided into three subarrays rather than two.
+As could be assumed, the sum of these subarrays is computed recursively as is done with mergesort's two subarrays. 
+
+The additional work noted to be c represents the constant time required to split the array into three subarrays and combine their sums, which remains the same constant work regardless of input size. 
+
+To solve this recurrence relation by substitution, we start with the above initial relation $T(n) = 3T(n/3) + c$.
+
+First, we substitute $T(n/3)$ with its own recurrence relation of $3T(n/9) + c$, in an attempt to break down the recurrence relation further (as desired for the substitution method).
+This gives us $T(n) = 3(3T(n/9) + c) + c$.
+Simplifying this, we get $T(n) = 9T(n/9) + 2c$.
+
+Here, it can be seen that the original problem of size n has been reduced to a problem of size n/9, with constant additional work 2c.
+The constant additional work comes from the c term in the original relation (representing the constant combination time of subproblem solutions).
+
+Next, we substitute $T(n/9)$ with its recurrence relation, again continuing to attempt to break down the problem and how the problem size performs and reduces exponentially.
+This results in $T(n) = 9(3T(n/27) + c) + 2c$.
+Simplifying further, we get $T(n) = 27T(n/27) + 3c$.
+
+At this point, the problem size has been reduced to $n/27$, with only constant additional work c accumulated.
+The work accumulates as i*c, with i being the current depth of recursion.
+
+Continuing this process, we can generalize the form of the recurrence relation based upon the behavior of the reduced equations above.
+
+After i substitutions, the recurrence relation will become $T(n) = 3^i T(n/3^i) + i * c$.
+To reach the base case T(1), we need $n/3^i = 1$.
+T(1) represents the constant work to handle the base case (returning a single element).
+Solving for i, i can be observed as $i = \log_3{n}$ in order to satisfy the statement.
+
+Substituting i back into the general recurrence form, we get $T(n) = n * T(1) + c * \log_3{n}$.
+Since T(1) is a constant, this can be further simplified into $T(n) = Θ(n) + Θ(\log n) = Θ(n)$.
```

Comment:
And please use the notation we've used in class to denote that a particular $T(n)$ expression is a member of a particular complexity here.